backend/models/base.py

```diff
@@ -131,6 +131,10 @@ def to_dict(self):
         schema = self.__marshmallow__()
         data = schema.dump(self)
         data["display_name"] = self.display_name
+        # add the transient attribute that tracks the change request responsible for changes (if it exists)
+        # so that it's added to the history event details
+        if hasattr(self, "acting_change_request_id"):
```

Comment:
This would be better on a SQLAlchemy model specific `to_dict()` rather than on the `BaseModel`

Reply:
I was thinking change requests could be used on any entity in the future.  For now it's just BLIs and next is Agreement if we do procurement shop changes.  But I can move to the BLI for now.

Reply:
done